core/templates/dev/head/pages/exploration_editor/editor_navbar_breadcrumb_directive.html

```diff
@@ -0,0 +1,20 @@
+<script type="text/ng-template" id="inline/editor_navbar_breadcrumb_directive">
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span ng-if="navbarTitle">
+        <[navbarTitle]>
+        <span class="oppia-navbar-breadcrumb-separator" style="margin-left: 5px;"></span>
+        <span>
+          <[getCurrentTabName()]>
+        </span>
+      </span>
+      <span ng-if="navbarTitle === ''" ng-click="editTitle()" style="opacity: 0.6;">
+        Untitled Exploration
+      </span>
+      <span ng-if="navbarTitle === null">
+        Loading...
+      </span>
+    </li>
+  </ul>
+</script>
```

Comment:
Please ensure that there is a newline at EOF, for this file and others. You can see whether it's correct by looking at https://github.com/oppia/oppia/pull/2844/files and ensuring that there's no "error" sign at the end of the file.